airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2615,6 +2615,7 @@ components:
           enum: ["oauth2.0"] # Future auth types should be added here
         oauth2Specification:
           "$ref": "#/components/schemas/OAuth2Specification"
+      nullable: true
```

Comment:
Can these `nullable: true` changes be removed now that the other API fix you added has been merged?